sklearn/manifold/_spectral_embedding.py

```diff
@@ -338,7 +372,13 @@ def spectral_embedding(
         X = random_state.standard_normal(size=(laplacian.shape[0], n_components + 1))
         X[:, 0] = dd.ravel()
         X = X.astype(laplacian.dtype)
-        _, diffusion_map = lobpcg(laplacian, X, M=M, tol=1.0e-5, largest=False)
+        # While scikit-learn has a minimum scipy dependency <1.4.0 we require
+        # high tolerance as explained in:
+        # https://github.com/scikit-learn/scikit-learn/pull/13707#discussion_r314028509
+        tol = 1e-5 if eigen_tol == "auto" else eigen_tol
```

Comment:
@lobpcg Is it better to hard code 1e-5 here or pass in `None`, so that SciPy can uses it's heuristic of `tol=n*sqrt(eps)` ?

https://github.com/scipy/scipy/blob/820fe86bf3ad8ba1c5647632b131659ed7bd447e/scipy/sparse/linalg/_eigen/lobpcg/lobpcg.py#L398